### DISCUSSION TITLE
Using shallow returns "Property X does not exist on type 'StoreApi<unknown>'.

### DISCUSSION THREAD

**Discussion by livingdotone** [pmndrs/zustand#1225] [Q&A]
![Captura de tela de 2022-08-25 11-01-42](https://user-images.githubusercontent.com/65691609/186685677-fc520293-1388-420d-9aeb-78ce706c969a.png)

Any help?

**Comment by dai-shi** [comment]
Can you create a reproduction with typescript playground https://www.typescriptlang.org/play, or codesandbox https://react.ts.new ?

**Reply by livingdotone** [reply]
Sorry, my import from shallow is wrong.
